### PR TITLE
Feature/auth page wording

### PR DESCRIPTION
**Pull request by jc-hiroto**
- Modified some icons and words.

### CODE REVIEW

**Comment by jc-hiroto** [comment]
@maxchou415 DONE.

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@nuxtjs/auth-next": "5.0.0-1622309801.87126e3",
     "@nuxtjs/axios": "^5.13.1",
+    "@nuxtjs/fontawesome": "^1.1.2",
```
Comment (maxchou415) [inline package.json:14]: 這個你有用到ㄇ

File: nuxt.config.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ export default {
     ],
     link: [
       { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' },
-      { rel: 'stylesheet', href: 'https://unpkg.com/tachyons@4.12.0/css/tachyons.min.css' }
+      { rel: 'stylesheet', href: 'https://unpkg.com/tachyons@4.12.0/css/tachyons.min.css' },
+      { rel: 'stylesheet', href: 'https://pro.fontawesome.com/releases/v5.10.0/css/all.css'}
```
Comment (maxchou415) [inline nuxt.config.js:22]: 還有這ㄍ

Comment (jc-hiroto) [reply nuxt.config.js]: removed

**Pull request merged**